pkgs/development/libraries/portaudio/default.nix

```diff
@@ -2,10 +2,11 @@
 , AudioUnit, AudioToolbox, CoreAudio, CoreServices, Carbon }:
 
 stdenv.mkDerivation {
-  name = "portaudio-190600-20161030";
+  pname = "portaudio";
+  version = "190600-20210406";
```

Comment:
The version should be `190700_20210406` (see the original `src.url` below).

Reply:
Yes, thank you. Sandro caught it a couple minutes ago : )

Reply:
I am keeping this conversation unresolved until it has been addressed in the commit.

Reply:
It was already addressed in [this commit](https://github.com/NixOS/nixpkgs/pull/126411/commits/eb3295d44c8940909350383c6f2b2188b6a8e0b5)

Reply:
No, you're right. I managed to mash up the old and new versions again!
Thanks!

Reply:
Confirmed that f24c0b3 is what I meant